### DISCUSSION TITLE
Genshin代理助手

### DISCUSSION THREAD

**Discussion by XoXing** [Melledy/Grasscutter#2086] [Support]
Genshin代理助手，在［_RUN SERVER (for java 17.0.6)］啟動後開始代理，卻顯示＂無法連接到遊戲服務器，可能是代理故障＂

![螢幕擷取畫面 2023-03-26 003507](https://user-images.githubusercontent.com/129092641/228001779-8298df1c-10f5-41d9-ae49-24e79e8fb803.png)

**Comment by ghost** [comment]


**Reply by XoXing** [reply]
很少接觸這種東西，不理解你所說的"Fiddler"是什麼

**Comment by NotThorny** [comment]
You can see the instructions for setting up Fiddler on the front page, specifically here: https://github.com/Grasscutters/Grasscutter#connecting-with-the-client
![firefox_WtyUeTWiJR](https://user-images.githubusercontent.com/107363768/228381141-a423853c-b31a-4afd-88f6-c823bea55a45.png)

The link to the script is on the front page, or you can go directly to the wiki to get it.
